logs_build/EventHubs/sumometricclient.js

```diff
@@ -0,0 +1,227 @@
+/*jshint esversion: 6 */
```

Comment:
As this is for logs, maybe it's better to remove this file to avoid confusion

Reply:
I also thought so first but in mainindex.js both sumometricclient and sumoclient are included.